devito/ir/iet/visitors.py

```diff
@@ -152,7 +152,9 @@ def _args_decl(self, args):
         """Generate cgen declarations from an iterable of symbols and expressions."""
         ret = []
         for i in args:
-            if i.is_Tensor:
+            if i.is_Tensor and i.is_OPS:
```

Comment:
at this point we shouldn't distinguish between different backends, otherwise it's gonna get incredibly messy as the number of backends and/or special cases (like this one) increases.

The rule of thumb is: if it's about OPS, then it should stay within `devito/ops`

Reply:
👍